apps/web/src/components/NotificationsModal/NotificationTwitterAlert.tsx

```diff
@@ -0,0 +1,86 @@
+import { useCallback } from 'react';
+import { graphql, useFragment } from 'react-relay';
+import styled from 'styled-components';
+
+import IconContainer from '~/components/core/IconContainer';
+import { ENABLE_TWITTER_DISMISSED_KEY } from '~/constants/storageKeys';
+import { TWITTER_AUTH_URL } from '~/constants/twitter';
+import { NotificationTwitterAlertFragment$key } from '~/generated/NotificationTwitterAlertFragment.graphql';
+import usePersistedState from '~/hooks/usePersistedState';
+import CloseIcon from '~/icons/CloseIcon';
+import InfoCircleIcon from '~/icons/InfoCircleIcon';
+
+import colors from '../core/colors';
+import InteractiveLink from '../core/InteractiveLink/InteractiveLink';
+import { HStack } from '../core/Spacer/Stack';
+import { BaseM } from '../core/Text/Text';
+
+type Props = {
+  queryRef: NotificationTwitterAlertFragment$key;
+};
+
+export function NotificationTwitterAlert({ queryRef }: Props) {
+  const query = useFragment(
+    graphql`
+      fragment NotificationTwitterAlertFragment on Query {
+        viewer {
+          ... on Viewer {
+            user {
+              socialAccounts {
+                twitter {
+                  __typename
+                }
+              }
+            }
+          }
+        }
+      }
+    `,
+    queryRef
+  );
+
+  const twitterAccount = query.viewer?.user?.socialAccounts?.twitter;
+
+  const [twitterDismissed, setTwitterDismissed] = usePersistedState(
+    ENABLE_TWITTER_DISMISSED_KEY,
+    false
```

Comment:
I think we can use the new experiences API. Check out `useExperience.ts`

Reply:
+1, we should definitely use experience flags. i'll record a loom on it!

Reply:
https://www.loom.com/share/97a3402a018941a491410278fdb789f7

Reply:
https://github.com/gallery-so/go-gallery/pull/827

Reply:
Updated to `useExperience`!